### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by karma4u101**
Added deps for for lift 3.1.1 / Scala 2.12.2 builds.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -2,19 +2,19 @@ name := "textile"
...
 
 moduleName <<= (name, liftEdition) { (n, e) =>  n + "_" + e }
 
-scalaVersion := "2.10.3"
+scalaVersion := "2.12.2"
 
 scalacOptions ++= Seq("-unchecked", "-deprecation")
 
-crossScalaVersions := Seq("2.11.1", "2.10.0", "2.9.2", "2.9.1-1", "2.9.1")
+crossScalaVersions := Seq("2.12.2", "2.11.1", "2.10.0", "2.9.2", "2.9.1-1", "2.9.1")
```
Comment (d6y) [inline build.sbt]: As there's no liftweb 3 for 2.10, I think we can drop 2.10 and 2.9

**Comment by karma4u101** [comment]
I dropped the 2.6 build. 
I deliberately kept the version matching in library dependencies, right now it is not necessary, but will most likely be in the future.

**Comment by fmpwizard** [comment]
Should we bump the version of this library? In case a user isbusing snapshot with scala 2.9

**Comment by karma4u101** [comment]
Maybe we need to (?) but I bumped it on my last update (a few days ago) and pushed a lift 3.1 snapshot. The last one I found, prior to my snapshot bump and release, was very strange, I don't remember exactly but it think it was something about the Lift and Scala versions that didn't match.

**Comment by karma4u101** [comment]
Upps I merged the wrong project / branch, was meen't to merge latest fontawesome in FoBo and opened the wrong tab ;( Let me know if there need to be some changes.

**Comment by d6y** [comment]
@karma4u101 :-) I was going to merge it in today and try to run a build of it anyway. Leave it with me and I'll hassle you if I run into weirdness :-)

**Comment by d6y** [comment]
Built fine for me, and there's a 1.4-SNAPSHOT version out to Sonatype

**Comment by fmpwizard** [comment]
sorry for the delay, if you already bumped it, sounds like we are all good now

**Pull request merged**